api/src/main/java/org/pmiops/workbench/db/dao/WorkspaceDao.java

```diff
@@ -164,4 +164,12 @@ interface WorkspaceCostView {
           + "WHERE w.activeStatus = 0 AND w.workspaceNamespace NOT IN (:referencedWorkspaceNamespaces)")
   List<String> findAllOrphanedWorkspaceNamespaces(
       @Param("referencedWorkspaceNamespaces") List<String> referencedWorkspaceNamespaces);
+
+  @Query(
```

Comment:
I thought @Query was heavily discouraged.... Can this be changed to call a stored-procedure or equivalent?

Reply:
ok thx.  I think we are good here.  As you say, this specific query has no parameters that could be injected.... and sounds like we have infranstructure protection as to when these are actually ran for ones that do have parameters.